### DISCUSSION TITLE
Load high resolution Images!

### DISCUSSION THREAD

**Discussion by shfomehr** [dotnet/maui#19619] [Q&A]
Hi
I'm new in .Net MAUI.
I want to add some high-resolution images in my app. The dimensions of the images are about 7000*4000 pixels, and they are layered on top of each other. (Layers will visible/invisible by buttons)
There is no problem on Windows platform, but when I run it on Android device or emulator, it runs very slow and laggy.
Considering that by double-clicking on the image, their scale will double or triple, if I reduce their dimensions, the final quality will decrease.
Is there a solution?

**Comment by xtuzy** [comment]
image is 7000*4000, but screen is little, and visiable region is little, you can use skiasharp load target region of image

**Comment by RobDaytona26** [comment]
I have the same issue, and yes the screen is little in comparison, but its for a map that sits inside a scroll view.. which should be possible, why cant it handle a 500K compressed jpg of these large dimensions, it just shows a black screen. Any work around?

**Comment by Xyncgas** [comment]
use the browser for anything content-performance related as adhoc solution

**Reply by RobDaytona26** [reply]
Sure, but in my case the map is a game map for a mobile game, so I can't get away from this problem being a problem, at this point I have settled on a smaller map, but its something I would like to see resolved, I think the cap is fairly arbitrary , I can't see any real technical limitations in 2024 and beyond for having such a small cap. Oh and I am invested in writing in Maui for mobile games, so no point trying to talk me out of it, already released my first game in Xamarin, and that went well.
